index.js

```diff
@@ -0,0 +1,42 @@
+var mocha = require('mocha');
+var stackFilter = require('mocha/lib/utils').stackTraceFilter();
+const VError = require("verror");
+
+function MochaVErrorReporter(runner)
+{
+  mocha.reporters.Base.call(this, runner);
+  runner.on('fail', function(test, err) {
+    err.stack = stackFilter(fullStack(err));
```

Comment:
@bgedik I used a mocha function for "prettifying" the stack output. It doesn't do much other than removing some internal calls